tests/dags/test_issue_2099.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use a descriptive name for the file and dag. This makes it easier to make out when it fails.